aspnetcore/fundamentals/localization/sample/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -1,14 +1,9 @@
-﻿<environment names="Development">
-    <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
-    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+﻿<environment include="Development">
+    <script src="~/js/jquery.validate-1.16.0.js"></script>
+    <script src="~/js/jquery.validate.unobtrusive-3.2.6.js"></script>
 </environment>
-<environment names="Staging,Production">
-    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js"
-            asp-fallback-src="~/lib/jquery-validation/dist/jquery.validate.min.js"
-            asp-fallback-test="window.jQuery && window.jQuery.validator">
-    </script>
-    <script src="https://ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js"
-            asp-fallback-src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js"
-            asp-fallback-test="window.jQuery && window.jQuery.validator && window.jQuery.validator.unobtrusive">
+<environment exclude="Development">
+    <script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.16.0/jquery.validate.min.js" asp-fallback-src="~/js/jquery.validate-1.16.0.min.js" asp-fallback-test="window.jQuery && window.jQuery.validator"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.6/jquery.validate.unobtrusive.min.js" asp-fallback-src="~/js/jquery.validate.unobtrusive-3.2.6.min.js" asp-fallback-test="window.jQuery && window.jQuery.validator && window.jQuery.validator.unobtrusive"></script>
     </script>
```

Comment:
Maybe I'm reading this wrong, but it looks like you have an extra script end-tag?

Reply:
Ah! I think ur correct. Thanks for spotting it. I'll fix it.